tutorials/katacoda/1-serving-intro/step2.md

```diff
@@ -0,0 +1,26 @@
+## Deploy and autoscale application
+
+We are going to deploy the [Hello world sample web application](https://knative.dev/docs/serving/samples/hello-world/helloworld-go/). This basic web application reads in an env variable TARGET and prints `Hello ${TARGET}!`. If TARGET is not specified, it will use `World` as the TARGET.
+
+We will now deploy the application by specifying the image location and the `TARGET` env variable.
+
+Knative defines a `service.serving.knative.dev` CRD to control the lifecycle of the application (not to be confused with kubernetes service). We will use the `kn` cli to create the Knative service:
+
+```
+kn service create demo --image gcr.io/knative-samples/helloworld-go --env TARGET=blue
+```{{execute}}
+
+We can now invoke the application using `curl`. We first need to figure out the IP address of minikube and the ingress port.
+```
+MINIKUBE_IP=$(minikube ip)
+INGRESS_PORT=$(kubectl get svc envoy --namespace contour-external --output 'jsonpath={.spec.ports[?(@.port==80)].nodePort}')
+```{{execute}}
+Then invoke the application using curl:
+```
+curl http://$MINIKUBE_IP:$INGRESS_PORT/ -H 'Host: demo.default.example.com'
+```{{execute T1}}
+
+### Scale down to zero
+You can run `watch kubectl get pods`{{execute T2}} in a new Terminal tab to see a pod created to serve the requests. Knative will scale this pod down to zero if there are no incoming requests for 60 seconds by default.
```

Comment:
When I click `watch kubectl get pods` It opens the terminal 2 in the right, side but nothing shows up.

Need to tell the user what to do to make the command output to show up
![image](https://user-images.githubusercontent.com/1094878/112038693-8f53a900-8b19-11eb-8373-f58c6999c58d.png)

Reply:
I ad to click twice, maybe one is to focus the T2 terminal and the second one is to run the command, not sure

Reply:
Yes, I have seen it sometimes too. It probably is a bug in katacoda, I added a note to click twice.